resources/filter_lines.rb

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright:: 2018 Sous Chefs
+# Copyright:: 2018, 2019 Sous Chefs
```

Comment:
should probably just be 2019?

Reply:
Parts are from 2018, parts are from 2019.  Only the parts from 2019 can claim a 2019 copyright.

Reply:
Fixed the dates.